### PR TITLE
Added code to perform read operation. Added testcases to test the read operation

### PR DESCRIPTION
**Pull request by veda10**
Added code to perform read operation. Added test cases to test the read operation.

##### SUMMARY

This patch helps in reading existing statistics and counters.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

Module: openvswitch_db
feature: read operation

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by veda10** [comment]
Hi @NilashishC  
I have been working on implementing the "read" operation in the ansible openvswitch_db module. I did this by adding another choice "read" in the state argument. I raised a PR but a check is failing "ansible-changelog-fragment check". Could you kindly help me to understand this check?
Thank you.

**Comment by NilashishC** [comment]
@veda10 Thank you for the pull request. Any user facing change in the code would require a changelog fragment. You can refer to this: https://github.com/ansible-collections/openvswitch.openvswitch/pull/66/files#diff-2421e2bca19aab45987cc3ca05305b003af079ea59df59f7fe720083895eb74cR1-R3.

**Comment by veda10** [comment]
@NilashishC Thank you for the suggestion. All the ansible/checks are passed now. But it says some checks haven't been completed yet which are of ansible/gate. Could you kindly help in understanding this? 
Thank you.

**Comment by NilashishC** [comment]
@veda10 The checks in the gate pipeline will start once a maintainer adds the `gate` label to this PR. This has been done to block manual merging of PRs. I have started reviewing your PR and once done, I shall add `gate` to this. Thanks!

File: plugins/modules/openvswitch_db.py
Comment (NilashishC) [inline plugins/modules/openvswitch_db.py:140]: Shouldn't this be `read`?

File: plugins/modules/openvswitch_db.py
Comment (NilashishC) [inline plugins/modules/openvswitch_db.py:145]: Same as above.

File: plugins/modules/openvswitch_db.py
Comment (veda10) [reply plugins/modules/openvswitch_db.py:140]: Yes. Initially, I used the word "get",  later changed it to "read". Missed updating that here. I will update it. Thanks

**Review by NilashishC — APPROVED** [review]

**Comment by NilashishC** [comment]
recheck

**Comment by NilashishC** [comment]
recheck

**Comment by NilashishC** [comment]
@veda10 We are currently experiencing some issues with our CI which is blocking this PR. Once the CI issues are resolved, this should be merged.

**Comment by NilashishC** [comment]
recheck

**Pull request merged**
